db-scripts/src/main/resources/migration.sql

```diff
@@ -409,7 +411,6 @@ CREATE TABLE `reference_genome_gene` (
     `EXONIC_LENGTH` int(11) DEFAULT NULL,
     `START` bigint(20) DEFAULT NULL,
     `END` bigint(20) DEFAULT NULL,
-    `ENSEMBL_GENE_ID` varchar(64) DEFAULT NULL,
```

Comment:
all changes in 2.4.1 done here should move to 2.7.0. 
I.e. the version 2.4.1 should remain as is. This means that we need to add the following to 2.7.0:
- truncate tables reference_genome_gene and reference_genome
- drop the column `ENSEMBL_GENE_ID`
- repeat the insert steps with the fixes made 

btw: the build name GRCh37 should perhaps also not be used hard coded here, but should rather come from portal.properties. This can be achieved by using a variable here, e.g. `${buildname}`. @sandertan will add some code to `migrate_db.py` (in this PR, if you are OK with that) to ensure this `buildname` variable is correctly replaced by the correct build name during migration execution in `migrate_db.py`.

Reply:
@pieterlukasse @sandertan , 
- No need truncate reference_genome_gene and reference_genome tables, instead create two tables only if they do not exists
- genome.name and genome.build have been dropped from the portal.properties as you suggested. You want to add them back?

Reply:
@pieterlukasse , can you please have a separate PR for your proposal for the migrate_db.py script, thanks!

Reply:
hi @khzhu, sure I'll make a PR to your branch, with an update to `migrate_db.py` and our suggestion for `migration.sql`. We still want to remove the 2 properties! Hopefully it'll be ready tomorrow, so you can review it.